wallet/src/de/schildbach/wallet/ui/WalletUriHandlerActivity.kt

```diff
@@ -49,11 +53,23 @@ class WalletUriHandlerActivity : AppCompatActivity() {
     }
 
     private var wallet: Wallet? = null
+    private lateinit var walletUriResultLauncher: ActivityResultLauncher<Intent>
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         wallet = (application as WalletApplication).wallet
         handleIntent(intent)
+        walletUriResultLauncher = registerForActivityResult(ActivityResultContracts.StartActivityForResult()) { result ->
+            var resultIntent: Intent? = null
+            if (result.resultCode == Activity.RESULT_OK) {
+                val data = result.data
+                val requestData = intent.data
+                val transactionHash = BitcoinIntegration.transactionHashFromResult(data)
+                resultIntent = WalletUri.createPaymentResult(requestData, transactionHash)
+            }
+            setResult(result.resultCode, resultIntent)
+            finish()
+        }
```

Comment:
replaces deprecated onActiivtyResult

Reply:
This is not used internally, but when apps externally request data using `dashwallet://...`